src/Components/FormComboBox.js

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { comboBox } from '../../node_modules/uswds/src/js/components';
+
+import { ValidationState } from '../Models/PatientFieldData';
+import ValidationMessage from './ValidationMessage';
+
+function FormComboBox({ label, property, required, onChange, options, showRequiredHint, size, validationState, value }) {
+  const ref = useRef();
+  const [focused, setFocused] = useState(false);
+  const [customOption, setCustomOption] = useState(null);
+
+  useEffect(() => {
+    const { current } = ref;
+    comboBox.on(current);
+    // manually add event handlers to the custom input added by USWDS
+    const input = current.querySelector('input[type="text"]');
+    input.addEventListener('input', (e) => setCustomOption(e.target.value));
+    input.addEventListener('focus', () => setFocused(true));
+    input.addEventListener('blur', () => setFocused(false));
+    input.value = value;
+    return () => {
+      comboBox.off(current);
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    const { current } = ref;
+    // manually add/remove validation classes to the custom input added by USWDS
+    const input = current.querySelector('input[type="text"]');
+    if (validationState === ValidationState.ERROR) {
+      input.classList.add('usa-input--error');
+    } else {
+      input.classList.remove('usa-input--error');
+    }
+    if (validationState === ValidationState.FIXED) {
+      input.classList.add('usa-input--success');
+    } else {
+      input.classList.remove('usa-input--success');
+    }
+  }, [validationState]);
```

Comment:
Changing the classes on the select element after comboBox initialization seemed to screw things up- so here I add another useEffect that watches the validationState and manually adds/removes the validation state from the custom input box added by the USWDS combobox component- again, we have to use standard DOM methods in this case...